chat_shell/chat_shell/tools/builtin/load_skill.py

```
@@ -103,51 +103,88 @@ def __init__(self, **data):
+            "[LoadSkillTool] Loaded skill '%s' (source=%s, remaining_turns=%d)",
             skill_name,
-            self.skill_retention_turns,
+            source,
+            turns,
         )
+        return True
+
+    def _run(
+        self,
+        skill_name: str,
+        run_manager: CallbackManagerForToolRun | None = None,
```

Ruff (0.14.14)

[warning] Unused method argument: `run_manager`

(ARG002)